--- a/rootfs/etc/xdg/autostart/lxqt-powermanagement.desktop
+++ b/rootfs/etc/xdg/autostart/lxqt-powermanagement.desktop
@@ -7,6 +7,7 @@ Exec=lxqt-powermanagement
 OnlyShowIn=LXQt;
 X-LXQt-Need-Tray=true
 X-LXQt-Module=true
+Hidden=true
 
 Name[da]=Strømstyring
 Name[de]=LXQt-Energieverwaltung
--- a/rootfs/etc/xdg/autostart/nm-applet.desktop
+++ b/rootfs/etc/xdg/autostart/nm-applet.desktop
@@ -3,6 +3,7 @@ Name=Network
 Comment=Manage your network connections
 Icon=nm-device-wireless
 Exec=nm-applet
+Hidden=true
 Terminal=false
 Type=Application
 NoDisplay=true
