--- a/rootfs/etc/xdg/pcmanfm/lubuntu/desktop-items-0.conf
+++ b/rootfs/etc/xdg/pcmanfm/lubuntu/desktop-items-0.conf
@@ -2,8 +2,8 @@
 wallpaper_mode=crop
 wallpaper_common=1
 wallpapers_configured=1
-wallpaper0=/usr/share/lubuntu/wallpapers/lubuntu-default-wallpaper.png
-wallpaper=/usr/share/lubuntu/wallpapers/lubuntu-default-wallpaper.png
+wallpaper0=/usr/share/lubuntu/wallpapers/leo-mars-wallpaper.jpg
+wallpaper=/usr/share/lubuntu/wallpapers/leo-mars-wallpaper.jpg
 desktop_bg=#2e4060
 desktop_fg=#ffffff
 desktop_shadow=#000000
--- a/rootfs/etc/xdg/pcmanfm/lubuntu/pcmanfm.conf
+++ b/rootfs/etc/xdg/pcmanfm/lubuntu/pcmanfm.conf
@@ -11,8 +11,8 @@ autorun=1
 wallpaper_mode=crop
 wallpaper_common=1
 wallpapers_configured=1
-wallpaper0=/usr/share/lubuntu/wallpapers/lubuntu-default-wallpaper.png
-wallpaper=/usr/share/lubuntu/wallpapers/lubuntu-default-wallpaper.png
+wallpaper0=/usr/share/lubuntu/wallpapers/leo-mars-wallpaper.jpg
+wallpaper=/usr/share/lubuntu/wallpapers/leo-mars-wallpaper.jpg
 desktop_bg=#2e4060
 desktop_fg=#ffffff
 desktop_shadow=#000000
