--- a/rootfs/boot/firmware/README
+++ b/rootfs/boot/firmware/README
@@ -18,9 +18,3 @@ follows:
                    loads...
 * vmlinuz        - the Linux kernel, executed by boot.scr
 * initrd.img     - the initramfs, executed by boot.scr
-* meta-data      - meta-data for cloud-init; usually just contains the
-                   instance id
-* network-config - network configuration for cloud-init; edit this to set up
-                   wifi access points and other networking settings
-* user-data      - user-data for cloud-init; edit this to configure initial
-                   users, SSH keys, packages, etc.
